.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+Hello! Thank you for reporting an issue!
+If you would fill out the below points, that would make our process a whole lot easier!
+
+## **I'm submitting a ...**
+  - [ ] bug report
+  - [ ] feature request
+  - [ ] support request
+
+
+
+
+## Regarding bug reports:
+
+* **Please tell us about your environment:** 
+  - Jrnl version: (run `jrnl -v`)
+  - Python version: (run `python -V`)
```

Comment:
Note that `jrnl` might be using a different python than the one that `python` points to. The quickest way to find it is to run ``head -1  `which jrnl` ``, but we should probably just print this when running `jrnl -v` instead.

Reply:
I'd argue that'd be a thing `jrnl -v` should print. i.e: `jrnl version 1.9.8 (running on python3.8.1)` kindof.

Reply:
I've added this to the `jrnl -v` output now :)